build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -207,6 +207,20 @@ service ActionCache {
 // `{instance}/uploads/{uuid}/blobs/{hash}/{size}/foo/bar/baz.cc`. Anything
 // after the `size` is ignored.
 //
+// Clients can upload compressed data with a `resource_name` of the form
+// `{instance_name}/uploads/{uuid}/compressed-blobs/{compressor}/{uncompressed_hash}/{uncompressed_size}{/optional_metadata}`
+// where `compressor` is the lowercase string form of a `Compressor.Value` enum
+// other than `identity` which is advertised by the server in
```

Comment:
Is this explicitly saying that 'identity' cannot be served over compressed-blobs? I think probably so--the 'rules' for uncompressed data are slightly different (e.g., in some cases using gRPC compression might make sense)--but want to be sure this is intentional.

Reply:
This is intentional, so clients don't try to use compressed-blobs resource names with old servers that do not populate CacheCapabilities.supported_compressor.